tsdef/LocalVideoTrack.d.ts

```diff
@@ -0,0 +1,21 @@
+import { LocalTrackOptions } from './LocalTrackOptions';
+import { Track } from './Track';
+import { VideoTrack } from './VideoTrack';
+
+export class LocalVideoTrack extends VideoTrack {
+  constructor(mediaStreamTrack: MediaStreamTrack, options?: LocalTrackOptions);
+
+  id: Track.ID;
+  isStopped: boolean;
+
+  disable(): LocalVideoTrack;
+  enable(enabled?: boolean): LocalVideoTrack;
+  restart(constraints?: MediaTrackConstraints): Promise<void>;
+  stop(): LocalVideoTrack;
+
+  on(event: 'dimensionsChanged', listener: (track: VideoTrack) => any): this;
```

Comment:
@PikaJoyce Please change return type of all event listeners to `void` (even the ones already committed).
```suggestion
  on(event: 'dimensionsChanged', listener: (track: VideoTrack) => void): this;
```